lib/diagnostic.js

```diff
@@ -60,11 +60,13 @@ let deathStar = {
 // Question 4: Write the code that would set the `deathStar`'s `fire` method
 // in the context of `alderaan` but does not invoke the method,
 // so that we could call answer4() // Alderaan
-let answer4 = null;  // replace null with your response
+let answer4 = function () {
+  return deathStar.fire.call(alderaan);  // replace null with your response
+};
```

Comment:
This technically works, because you made answer4 a function that returns the correct string.

`let answer4 = deathStar.fire.bind(alderaan);`

All you needed to was bind alderaan as we didn't want to invoke the fire method, we just wanted to set the context so that `answer4()` would invoke fire on alderaan.